### ISSUE TITLE
Product image block image is not displayed in the frontend but in the backend.

### ISSUE THREAD

**Issue by codestylist** [woothemes/woocommerce#50755]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

I'm facing an issue with the product image block (`woocommerce/product-image`) which is included in the single product block (`woocommerce/single-product`). The image is not displayed in the frontend but in the backend.
The code is "empty". It does not have the image.
`<div data-block-name="woocommerce/product-image" data-show-product-link="false" data-show-sale-badge="false" data-sale-badge-align="left" data-image-sizing="thumbnail" data-width="" data-height="" class="is-loading"></div>`

### Expected behavior

I would expect the image to be displayed in the frontend in the same way as in the backend (when I edit the page with the block editor).

### Actual behavior

When I add the product image block within the single product block in the backend (block editor) the image is shown. But when I open the page in the front end, the image is missing.

### Steps to reproduce

1. create a WordPress instance.
2. activate any Twenty theme (for example TwentyTwenty One).
3. install and activate WooCommerce.
4. create a product and make sure to add a product image.
5. create a new page and add the single product block.
6. select the previously created product.
7. save the page.
8. display the page in the preview.

### WordPress Environment

WordPress 6.6.1
WooCommerce 9.1.4
TwentyTwenty Two theme

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by albarin** [comment]
Hi @codestylist!
Thanks for reporting this issue. We followed the steps you provided but were not able to reproduce the issue. The image is shown in the editor and also in the front end.
Could you provide some more details? (like screenshots or a video so we can see exactly what's going on).
Thanks a lot!

**Comment by codestylist** [comment]
Thank you @albarin for taking the time to look into this issue. I was surprised by your response and set up a new staging site to test. And voila, it works. So I went back to my previous test and it didn't work. I created a new site there, added the block and it works too. So I deleted the initial block and created a new one on the original page. It works now too. The thing I can imagine is that the very first block with this problem was created with an earlier version of WooCommerce 8.x. The code of the block was probably not updated after the update to WooCommerce 9.
